server/metrics/metrics.go

```diff
@@ -251,6 +257,41 @@ var (
 	/// sum(rate(buildbuddy_remote_execution_count[5m]))
 	/// ```
 
+	/// #### Examples
+	///
+	/// ```promql
+	/// # Median action queue duration
+	/// histogram_quantile(
+	///	  0.5,
+	///   sum(rate(buildbuddy_remote_execution_queue_duration_usec_bucket[5m])) by (le)
+	/// )
+	/// ```
+
+	RemoteExecutionCommandPhaseDurationUsec = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
How about calling this something related to ExecutedActionMetadata and only using those 5 phases here? That way we kind of don't go too crazy with this but it's still useful to get high-level execution stats and it's clear to the reader where this data came from / that it maps to this existing bazel concept?

Reply:
Good point -- done